azavea/azavea-branding-guide
# Make site responsive, update READme, and general cleanup

## Overview
This PR makes the website fully responsive (JS was broken previously and pages looked broken at smaller screen widths). At the same time, I updated the READme to include more detailed information about the breakdown of the pages and completed the following clean up tasks:
- Removed files and references to StyleGuideGuide boilerplate code
- Changed style of Logo Don'ts blocks
- Moved most of the pages into a `_pages` folder 
- Updated some boilerplate copy in the Release History message 

### Demo
Mobile menu
![screen shot 2018-07-18 at 11 57 36 am](https://user-images.githubusercontent.com/5672295/42898969-a6631a0c-8a92-11e8-94ab-b2dd59a67943.png)
![screen shot 2018-07-18 at 12 06 13 pm](https://user-images.githubusercontent.com/5672295/42898983-b1317ca8-8a92-11e8-9d89-b3f4997a954b.png)

New Don'ts
![screen shot 2018-07-18 at 12 06 25 pm](https://user-images.githubusercontent.com/5672295/42898990-b7416dce-8a92-11e8-8e6a-30bc096172ae.png)
![screen shot 2018-07-18 at 12 06 29 pm](https://user-images.githubusercontent.com/5672295/42898991-b75045ec-8a92-11e8-9b5c-a68e8030a584.png)

Responsive colors section
![screen shot 2018-07-18 at 12 06 40 pm](https://user-images.githubusercontent.com/5672295/42899019-cd6978c6-8a92-11e8-8276-aaf9f3035060.png)
<img width="350" alt="screen shot 2018-07-18 at 12 07 15 pm" src="https://user-images.githubusercontent.com/5672295/42899020-cd75ab8c-8a92-11e8-9e28-b642d177d2a1.png">

### Notes
- `index.html`, `release-history.md`, `welcome.md` didn't seem to work when put into the _pages/ folder, so they're in the root
- The homepage intentionally does not have a "Menu" button.


## Testing Instructions
* `git pull`
* Go to the `README.md` file to see updates there.
* Play with the screen width on various pages to test the responsivity. The ones with the most changes were: the homepage, Colors page and Logo page (where you can also see the new "dont's" cards).
* Review the copy in Release History page
* Review the diff to see what files have been removed.

Closes #53, closes #10, closes #36

fyi @danford1
